Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -1011,8 +1011,15 @@ void CGameListCtrl::OnDeleteISO(wxCommandEvent& WXUNUSED (event))
 void CGameListCtrl::OnProperties(wxCommandEvent& WXUNUSED (event))
 {
 	const GameListItem* iso = GetSelectedISO();
-	if (!iso)
+	DiscIO::IVolume* OpenISO = DiscIO::CreateVolumeFromFilename(iso->GetFileName());
```

Comment:
Could you use a unique pointer here, so it gets automatically deleted at the end of the function.